pyproject.toml

```diff
@@ -0,0 +1,84 @@
+# @ pyproject.toml specifies how to install your package, and ensures it is
+# @ installed as a valid GeoIPS plugin package.
+
+# @ Replace all instances of @ within this template with appropriate values for
+# @ your particular use case.
+# @ Once complete, remove all lines containing "@" from this file.
+
+[build-system]
+requires = ["setuptools>=61.2", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages]
+find = {namespaces = false}
+
+[tool.setuptools.package-dir]
+"" = "."
+
+[tool.setuptools.package-data]
+"*" = [
+    "plugins/*.yaml",
+    "plugins/*/*.yaml",
+    "plugins/*/*/*.yaml",
+    "plugins/*/*/*/*.yaml",
+    "plugins/*/*/*/*/*.yaml",
+    "plugins/*.txt",
+    "plugins/*/*.txt",
+    "plugins/*/*/*.txt",
+    "plugins/*/*/*/*.txt",
+    "plugins/*/*/*/*/*.txt",
+]
+
+[tool.setuptools.dynamic]
+entry-points = {file = ["entry-points.ini"]}
+
+[tool.setuptools_scm]
+"version_scheme" = "post-release"  # Use current version .postN vs incrementing
+"local_scheme" = "no-local-version"  # Don't include hash, or date, just postN
+# @ Update with your package name, ie:
+# @ "write_to" = "@package@/version.py"  # Write an actual file
+"write_to" = "my_package/version.py"  # Write an actual file
+
+[project]
+# @ Update with your package name, ie:
+# @ name = "@package@"
+name = "my_package"
```

Comment:
I made this fully installable and runnable, so included comments annotated with "@" explaining what needs to be updated.

Reply:
Yeah, I like the new layout. It makes a lot of sense.